src/renderer/components/Editor/EditMarker.tsx

```diff
@@ -37,14 +26,19 @@ const EditMarker = ({ transcription, word, index, isSelected }: Props) => {
 
   const notPasted = word.pasteKey === 0;
 
-  const EditMarkerContainer = makeEditMarkerContainer(isSelected);
-
   return (isInOriginalPos || hasNotMoved) &&
     hasNoNeighbourMarker &&
     notPasted ? (
-    <EditMarkerContainer>
+    <Box
+      sx={{
```

Comment:
Any reason you're inlining these styles now?

Reply:
I could not get the onClick to work with the previous implementation. I think because the MUI component was converted to a React Component, the onClick that comes with a Box component was lost. I'd have to pass the onClick down through props, so I felt inline was the easiest and cleanest way.

Reply:
Interesting, no problem then